File: data/code_pairs/fortran/156.f90
program prog
  complex :: zero, one_two, three_four
  real, parameter :: three=3.0
  real, parameter :: four=three+1.0
  parameter(zero=(0.0e+0,0.0e+0))
  parameter(one_two=(1.0e+0,2.0e+0), three_four=(three,four))
  
  write(*, *) zero
  write(*, *) one_two
  write(*, *) three_four
end program prog

File: data/code_pairs/cpp/156.cpp
#include <iostream>
#include <complex>

int main() {
    std::complex<double> zero(0.0, 0.0);
    std::complex<double> one_two(1.0, 2.0);
    double three = 3.0;
    double four = three + 1.0;
    std::complex<double> three_four(three, four);

    std::cout << zero << std::endl;
    std::cout << one_two << std::endl;
    std::cout << three_four << std::endl;

    return 0;
}